database/sqlite/sqlite_health.c

```diff
@@ -0,0 +1,1357 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "sqlite_health.h"
+#include "sqlite_functions.h"
+
+#define MAX_HEALTH_SQL_SIZE 2048
+
+/* Health related SQL queries
+   Creates a health log table in sqlite, one per host guid
+*/
+#define SQL_CREATE_HEALTH_LOG_TABLE(guid) "CREATE TABLE IF NOT EXISTS health_log_%s(hostname text, unique_id int, alarm_id int, alarm_event_id int, config_hash_id blob, updated_by_id int, updates_id int, when_key int, duration int, non_clear_duration int, flags int, exec_run_timestamp int, delay_up_to_timestamp int, name text, chart text, family text, exec text, recipient text, source text, units text, info text, exec_code int, new_status real, old_status real, delay int, new_value double, old_value double, last_repeat int, class text, component text, type text);", guid
+int sql_create_health_log_table(RRDHOST *host) {
+    int rc;
+    char *err_msg = NULL, command[MAX_HEALTH_SQL_SIZE + 1];
+
+    char uuid_str[GUID_LEN + 1];
+    uuid_unparse_lower(host->host_uuid, uuid_str);
+    uuid_str[8] = '_';
+    uuid_str[13] = '_';
+    uuid_str[18] = '_';
+    uuid_str[23] = '_';
+
+    snprintfz(command, MAX_HEALTH_SQL_SIZE, SQL_CREATE_HEALTH_LOG_TABLE(uuid_str));
+
+    if (unlikely(!db_meta)) {
```

Comment:
if the test needs to exist it is better to be at the very top

Reply:
Ok, thanks! Done, moved.